gateway-service-definitions/src/main/resources/services/iceberg-rest/0.0.1/rewrite.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<rules>
+    <rule dir="IN" name="ICEBERG-REST/iceberg-rest/inbound" pattern="*://*:*/**/iceberg-rest/{path=**}?{**}">
```

Comment:
Looks like the service name does not have`api`, we have rest in the name but can we change it to`api` to be in line with other services e.g. `ICEBERG-API`

Reply:
We need REST in the name since it is a specific API. ICEBERG-REST-API is fine if we are using that convention.

Reply:
Right, and I'm not convinced there is really a convention to append -api as I review the existing service defs. I propose leaving it as it is, but I'm open to different opinions.